tests/src/test/java/alluxio/hadoop/FileSystemStatisticsTest.java

```diff
@@ -34,9 +34,10 @@
 
   private static final int BLOCK_SIZE = 128;
   private static final int FILE_LEN = BLOCK_SIZE * 2 + 1;
+  private static final int WORKER_CAPACITY_BYTES = 10000;
   @ClassRule
   public static LocalAlluxioClusterResource sLocalAlluxioClusterResource =
-      new LocalAlluxioClusterResource(10000, BLOCK_SIZE);
+      new LocalAlluxioClusterResource(WORKER_CAPACITY_BYTES, BLOCK_SIZE);
```

Comment:
hm...It looks like there is no int KB in Constants.class. And Constants.class cannot be changed(in a .jar file).  Do I need to import other library for Constant.class? Or should I just create an  Contans_KB = 1000 in FileSystemStatisticsTest and use it?@ifcharming